### PR TITLE
pass team name as argument

### PR DESCRIPTION
**Pull request by bullybutcher**

### CODE REVIEW

File: notebook/notebook.tex [hunk 1]
```diff
@@ -81,7 +81,11 @@
 { \end{itemize}                  }
 
 % Title/Author
-\title{$KFC$}
+\ifdefined\team 
+  \title{\team{}}
+\else
+  \title{$KFC$}
```
Comment (leloykun) [inline notebook/notebook.tex:87]: set the default team name to `Ateneo ProgVar`

**Review by leloykun — CHANGES_REQUESTED** [review]

File: notebook/makefile.sh [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
 
-pdflatex -shell-escape -jobname="notebook-trunc" notebook.tex
-pdflatex -shell-escape -jobname="notebook-trunc" notebook.tex
+pdflatex -shell-escape -jobname="notebook-trunc" "\def\ICPCCONFIG{1} \def\team{$teamname} \input{notebook.tex}"
+pdflatex -shell-escape -jobname="notebook-trunc" "\def\ICPCCONFIG{1} \def\team{$teamname} \input{notebook.tex}"
 
-pdflatex -shell-escape "\def\ICPCCONFIG{1} \input{notebook.tex}"
-pdflatex -shell-escape "\def\ICPCCONFIG{1} \input{notebook.tex}"
+pdflatex -shell-escape "\def\ICPCCONFIG{1} \def\team{$teamname} \input{notebook.tex}"
+pdflatex -shell-escape "\def\ICPCCONFIG{1} \def\team{$teamname} \input{notebook.tex}"
```
[the remark this answers is not in the document]
Comment (bullybutcher) [reply notebook/makefile.sh:16]: i think underscore `_` in latex is a reserved character

[the remark this answers is not in the document]
Comment (leloykun) [reply notebook/makefile.sh:16]: ah, nvm, let's just use `TEAMNAME` then

**Review by leloykun — APPROVED** [review]

**Comment by leloykun** [comment]
@bullybutcher , I'm just gonna merge this since it's been 6 days since I approved it

**Pull request merged**